AIDriver.lua

```diff
@@ -201,25 +201,42 @@ end
 
 function AIDriver:writeUpdateStream(streamId, connection, dirtyMask)
 	self.triggerHandler:writeUpdateStream(streamId)
-	streamWriteString(streamId,self.state.name)
-	if self.active then 
+	if self.state ~= self.stateSend then 
```

Comment:
can we call these stateSent (or lastStateSent) (in past tense) to make it clear what it is